packages/neos-ui-editors/src/SecondaryEditors/ImageCropper/index.js

```diff
@@ -86,11 +86,12 @@ export default class ImageCropper extends PureComponent {
     componentDidMount() {
         //
         // Calculate and set maximum height for the cropped image
-        const containerHeight = this.containerNode.parentElement.clientHeight;
+        const secondaryEditorHeight = window.innerHeight - 41 - 41;
```

Comment:
While this solution is a little hacky (as that we hardcode where it will be used) but i think this is fine for the current usecase?

Another solution would be to use bare css to fit the react cropper somehow. Or is there a way to get the minHeight of a container - the height it would have without any children (without hiding its contents)

Reply:
Just tried all my css wizardry but in the version we've got, there is no way (at least for me) we need to use max-width with hard coded values.

but also note we are using 1.2 while version !!! 10 !!! is out there - might as well be fixed now

Reply:
Add a comment (in the code), why we hardcoded it like this.
Then it's absolutely fine for me 👍